source/opt/strip_debug_info_pass.cpp

```diff
@@ -22,10 +22,19 @@ Pass::Status StripDebugInfoPass::Process() {
   bool modified = !context()->debugs1().empty() ||
                   !context()->debugs2().empty() ||
                   !context()->debugs3().empty();
+  for (auto& dbg : context()->debugs1()) context()->KillNamesAndDecorates(&dbg);
+
+  for (auto& dbg : context()->debugs2()) context()->KillNamesAndDecorates(&dbg);
+
+  for (auto& dbg : context()->debugs3()) context()->KillNamesAndDecorates(&dbg);
+
   context()->debug_clear();
 
-  context()->module()->ForEachInst([&modified](Instruction* inst) {
+  context()->module()->ForEachInst([&modified, this](Instruction* inst) {
     modified |= !inst->dbg_line_insts().empty();
+    for (auto& dbg : inst->dbg_line_insts())
+      this->context()->KillNamesAndDecorates(&dbg);
```

Comment:
OpLine instructions do not have a result id, so cannot have names or decorations.

Reply:
Done